lib/typescript/websocketclient/payload.ts

```diff
@@ -1,46 +1,47 @@
-import {DeliveryState, Metadata, MetadataEvent} from 'model';
+import {DeliveryState, Metadata, MetadataEvent, Tag} from 'model';
 
 interface Event {
-  type: 'message' | 'channel' | 'metadata';
-}
-
-interface MessagePayload {
-  id: string;
-  content: string;
-  delivery_state: DeliveryState;
-  from_contact: boolean;
-  sent_at: Date;
-  metadata: any;
-}
-
-interface ChannelPayload {
-  id: string;
-  metadata?: Metadata & {
-    name: string;
-    image_url?: string;
-  };
-  source: string;
-  source_channel_id: string;
-  connected: boolean;
+  type: 'message' | 'channel' | 'metadata' | 'tag';
 }
 
 export interface MessageEventPayload extends Event {
   type: 'message';
   payload: {
     conversation_id: string;
     channel_id: string;
-    message: MessagePayload;
+    message: {
+      id: string;
+      content: string;
+      delivery_state: DeliveryState;
+      from_contact: boolean;
+      sent_at: Date;
+      metadata: any;
```

Comment:
nitpick: Message events don't include metadata.

Reply:
yeah this was already there bu I agree. I'll change!